test/backend/CMakeLists.txt

```diff
@@ -15,5 +15,11 @@ add_custom_target(check-onnx-backend
         COMMAND ${PYTHON_EXECUTABLE}
         ${CMAKE_CURRENT_BINARY_DIR}/test.py)
 
+add_custom_target(clean-onnx-backend
```

Comment:
Is the cleanup called when doing a new "full" check-onnx-backend? Otherwise, we may have mixures of new and old files.

Reply:
For each test, the temporary files will be deleted before the new test. This is done in the test.py.  I  did not clean the whole directory in case someone need some previous result.